src/lib.rs

```diff
@@ -821,8 +821,12 @@ impl Config {
         cmd.arg("--config").arg(&profile);
 
         if let Ok(s) = env::var("NUM_JOBS") {
-            // See https://cmake.org/cmake/help/v3.12/manual/cmake.1.html#build-tool-mode
-            cmd.arg("--parallel").arg(s);
+            if cmake_has_parallel(&executable) {
+                cmd.arg("--parallel").arg(s);
+            } else {
+                println!("cargo:warning=The parallel flag wasn't passed to CMake!");
```

Comment:
Could the be a bit more conditional where this is only printed if `NUM_JOBS` is greater than one, there's no `Makefile`, and could the cmake version be printed here? (noting that the minimum version is 3.12.0)